docusaurus/docs/Android/03-ui/04-message-components/01-message-list-screen.mdx

```diff
@@ -61,8 +61,11 @@ class MyMessageListActivity : AppCompatActivity(R.layout.fragment_container) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         if (savedInstanceState == null) {
+            val fragment = MessageListFragment.newInstance(cid = "messaging:123") {
+                showHeader(true)
```

Comment:
Hidden by default, we need to set this to "true" to be on par with the example above.